sdk/nodejs/tests/runtime/closure.spec.ts

```diff
@@ -5040,6 +5041,49 @@ return function () { const v = new pulumi.Config("test").get("TestingKey2"); con
         });
     }
 
+    {
+        cases.push({
+            title: "Capture factory func #1",
+            factoryFunc: () => {
+                const serverlessExpress = require("aws-serverless-express");
+                const express = require("express");
+                const app = express();
+                app.get("/", (req: any, res: any) => {
+                    res.json({ succeeded: true });
+                });
+
+                const server = serverlessExpress.createServer(app);
+
+                return (event: any, context: any) => {
+                    serverlessExpress.proxy(server, event, context);
+                };
```

Comment:
this is the code that then becomes the exported 'handler' function for the module.  It can now use all those values which were only instantiated once.